regulations/static/regulations/js/source/views/main/preamble-view.js

```diff
@@ -38,6 +43,7 @@ var PreambleView = ChildView.extend({
     this.listenTo(MainEvents, 'paragraph:active', this.handleParagraphActive);
 
     CommentEvents.trigger('comment:readTabOpen');
+    DrawerEvents.trigger('pane:init', type === 'preamble' ? 'table-of-contents' : 'table-of-contents-secondary');
```

Comment:
I could imagine DrawerEvents like these being driven by the DOM, too, rather than trying to parse out the state in JS.